### ISSUE TITLE
User code is outputted with extra newlines (on Windows)

### ISSUE THREAD

**Issue by papademos** [Genivia/RE-flex#72]
User code written inside `%{ %}` or `%top{ }` outputs extra newlines.
This breaks multiline #defines, e.g:
```
#define FOOBAR(a, b) \
FOO(a); \
BAR(b);
```
...gives the output:
```
#define FOOBAR(a, b) \

FOO(a); \

BAR(b);
```

Not sure if relevant, but these are the arguments I used when running reflex 2.0.0 WIN.
```
win64\reflex.exe
--bison-complete
--bison-cc-parser=Parser 
--bison-cc-namespace=clax 
--namespace=clax 
--bison-locations 
--outfile=rscanner.cpp 
--header-file=rscanner.h 
scanner.l
```

**Comment by genivia-inc** [comment]
Thanks for the feedback!

That's peculiar.

The problem appears to be caused by the CRLF output that `reflex` produces on Windows.

I will turn that off. Producing LF works just fine as the file isn't saved in "binary mode" any way.

**Comment by genivia-inc** [comment]
Fixed in v2.0.1.

**Comment by papademos** [comment]
Thanks for the quick fix.
It makes sense as writing in text mode on windows replaces LF with CR LF, thus causing a manual CR LF to be written as CR CR LF, hence leaving the interpretation of CR CR LF to the consuming application, in this case the compiler.
